package.json

```diff
@@ -40,7 +40,6 @@
     "istanbul-lib-coverage": "^3.2.0",
     "istanbul-lib-report": "^3.0.1",
     "istanbul-reports": "^3.1.6",
-    "rimraf": "^3.0.2",
```

Comment:
In package.json, update the engines field to `>=14.14.0` this will help catch folks who try to use versions of Node without `rm`.